fvm/crypto/crypto.go

```diff
@@ -37,27 +36,23 @@ func (DefaultSignatureVerifier) Verify(
 	publicKey crypto.PublicKey,
 	hashAlgo hash.HashingAlgorithm,
 ) (bool, error) {
+
 	var hasher hash.Hasher
 
 	switch hashAlgo {
 	case hash.SHA2_256:
-		hasher = hash.NewSHA2_256()
+		fallthrough
 	case hash.SHA3_256:
-		hasher = hash.NewSHA3_256()
-	case hash.SHA2_384:
```

Comment:
as far as I can see `hash.SHA2_384` and `hash.SHA3_384` go to the default (error) case now, it this ok?

Reply:
Yes, both `384` versions will not be involved for signature verifications. 
There is also a an extra compatibility check of what hashing algo is compatible with what signature algo that we will enforce in the signature branch.